contracts/tasks/vault.js

```diff
@@ -40,6 +40,15 @@ async function getContracts(hre, symbol, assetSymbol) {
   const oToken = await ethers.getContractAt(symbol, oTokenProxy.address);
   log(`Resolved ${network} ${symbol} OToken to address ${oToken.address}`);
 
+  // Resolve the wrapped OToken. eg wOETH, wOSonic
+  const wOTokenProxy = await ethers.getContract(
```

Comment:
Can we instead have a mapping with hardcoded names with Network IDs as keys? Would make it more obvious during searches. Also, we can easily accommodate any contracts that don't follow this naming convention

Reply:
We can, but it means devs have to remember to add to the mapping.
I suspect the next chain we add will break the convention and we'll have to use a mapping. Let's do that change when we need to so we don't need to do mapping maintenance.

Reply:
That sounds good, thanks